buoi_7_js/btvn_2.js

```diff
@@ -0,0 +1,18 @@
+/* Bài 2: 
+-	Input: Cho 1 object
+-	Output: Viết hàm kiểm tra xem object đó rỗng không */
+
+//Bài làm
+//const car = {type:"Fiat", model:"500", color:"white"};
+//console.log( Object.keys(car).length) => KQ 3  ( Object.values() tương tự)
+
+function checkNone(obj ={}) {
```

Comment:
- sao không return hàm
- Check None của cái gì thế??? => đặt lại tên hàm => Sao không đặt là checkObjNone hay là checkEmptyOfObj ...something like that